### ISSUE TITLE
GitHub auth.json über Manager generieren [Feature]

### ISSUE THREAD

**Issue by contaoacademy** [contao/contao-manager#535]
**Fehlermeldung**
`Could not authenticate against github.com`

**Lösung**
GitHub > Developer-Settings -> Personal Access Token
`auth.json` anlegen und in /contao-manager speichern

**Feature**
Es wäre hilfreich, wenn der AccessToken direkt über den Contao Manager hinterlegt werden könnte.

**Comment by frontendschlampe** [comment]
Was noch zusätzlich als Meldung kommt:

```html
Installing php-http/guzzle6-adapter (v1.1.1): Downloading    Failed to download php-http/guzzle6-adapter from dist: Could not authenticate against github.com
    Now trying to download from source
```

**Comment by iGweb** [comment]
Hello,

I have exactly the same error message as you do on an installation.
Unable to install contao via manager.
I get the following error message: "Could not authenticate against github.com"

Have you found a solution to the problem?

Thank you

**Comment by Toflar** [comment]
GitHub has admitted that they deployed changes. See https://github.com/composer/composer/issues/8710.
I expect things to get back to normal soon.

**Comment by iGweb** [comment]
Hello,

Thank you for your reply.
Until this returns to normal, is there a temporary solution to be able to install modules with contao-manager?

Thanks for your help

**Comment by contaoacademy** [comment]
> Hello,
> 
> Thank you for your reply.
> Until this returns to normal, is there a temporary solution to be able to install modules with contao-manager?
> 
> Thanks for your help

https://contao-academy.de/blog/authentifizierung-github-access-token-contao-manager

**Comment by iGweb** [comment]
Merci :)

**Comment by contaoacademy** [comment]
https://developer.github.com/changes/2020-02-10-deprecating-auth-through-query-param/

Ist dieser Hinweis relevant?
Müsste hier etwas am Manager geändert werden?

**Comment by aschempp** [comment]
Nein
1: der Manager greift nie selber auf GitHub zu, das macht alles Composer
2: ich wage zu behaupten dass nie Query Auth genutzt wurde

**Comment by Toflar** [comment]
Doch, das wurde in Composer 1.9.3 im Januar gefixt: https://github.com/composer/composer/commit/460c673cc40435b3e5def2b3827f3ca0dcead1db
